maven-release-manager/src/main/java/org/apache/maven/shared/release/phase/ScmTagPhase.java

```diff
@@ -168,6 +171,10 @@ public ReleaseResult simulate( ReleaseDescriptor releaseDescriptor, ReleaseEnvir
             logInfo( result, "Full run would be tagging remotely " + basedirAlignedReleaseDescriptor.getScmSourceUrl()
                 + " with label: '" + releaseDescriptor.getScmReleaseLabel() + "'" );
         }
+        if ( releaseDescriptor.isPinExternals() )
+        {
+            logInfo( result, "Full run would pin externals" );
```

Comment:
This should be consistent with the other log statement

Reply:
Fixed.